packages/components/accordion/src/accordion-item.ts

```diff
@@ -0,0 +1,119 @@
+import { localized } from '@lit/localize';
+import { type EventEmitter, event } from '@sl-design-system/shared';
+import { type CSSResultGroup, LitElement, type TemplateResult, html } from 'lit';
+import { property, query } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import styles from './accordion-item.scss.js';
+
+let nextUniqueId = 0;
+
+/**
+ * An accordion item component.
+ *
+ * @slot default - Body content for the accordion
+ * @part summary - Header element of the accordion-item
+ */
+@localized()
+export class AccordionItem extends LitElement {
+  /** @private */
+  static override styles: CSSResultGroup = styles;
+
+  /**
+   * Unique ID for each accordion item component.
+   */
+  #accordionItemId = `sl-accordion-item-${nextUniqueId++}`;
+
+  /**
+   * Unique ID for each accordion item component content.
+   */
+  #accordionItemContentId = `sl-accordion-item-content-${nextUniqueId++}`;
+
+  /** A text shown in the header - as a title of the accordion item. */
+  @property() summary!: string;
```

Comment:
Why the exclamation mark here? In other places we just do `summary?: string`

Reply:
Because summary should be always added, so I thought that we can assume that it's always there.

Reply:
We can't really. We can't force the user to set the `summary` property. In some places you can assume that something is set, but in this case it's dangerous.